config/consensus.go

```diff
@@ -446,6 +446,12 @@ type ConsensusParams struct {
 	// available for lookup for smart contracts and smart signatures.
 	// Setting it to 1 for example allows querying data up to MaxTxnLife + 1 rounds back from the Latest.
 	DeeperBlockHeaderHistory uint64
+
+	// ClearOfflineStateProofID tells the ledger to clear the account's StateProofID when a keyreg transaction marks it offline.
+	ClearOfflineStateProofID bool
```

Comment:
no need to have both parameters only `EnableOnlineAccountCatchpoints` is enough . The idea is eventually all nodes will have it clear. Check this logic:
1. node A upgrades before the new consensus takes effect. It has DB with clear StateProofID, it clears StateProofID in new transactions and does not generate catchpoints until EnableOnlineAccountCatchpoints sets to ON.
2. node B upgrades after the consensus. It has DB with clear StateProofID, it clears StateProofID in new transactions and generates catchpoints.